### PR TITLE
Implement MAB's UserActivityTracker and blocking the bridge upon exceeding the user limit

### PR DESCRIPTION
**Pull request by tadzik**
Developed in tandem with https://github.com/matrix-org/matrix-appservice-bridge/pull/350

### CODE REVIEW

File: config.sample.yaml [hunk 1]
```diff
@@ -556,6 +556,9 @@ ircService:
     # how much time needs to pass between the reply and the original message to switch to the long format
     shortReplyTresholdSeconds: 300
 
+  # Maximum number of montly active users, beyond which the bridge gets blocked (both ways)
+  # RMAUlimit: 100
```
Comment (Half-Shot) [inline config.sample.yaml:560]: Needs a JSON schema change too :stuck_out_tongue:

**Review by Half-Shot — APPROVED** [review]
No concerns about this one

**Comment by Half-Shot** [comment]
I believe we need #1453 for this to work?

**Comment by tadzik** [comment]
> I believe we need #1453 for this to work?

I guess so – though locally I've also backported the MAB PR to the 2.6.1 that IRC bridge currently relies on, and it works too – we can use that if we're in a hurry.

File: config.sample.yaml (hunk 1 above)
Comment (tadzik) [reply config.sample.yaml:560]: Done in 26505376 :)

**Review by Half-Shot — APPROVED** [review]

**Pull request merged**